css/src/components/message.scss

```diff
@@ -58,4 +60,16 @@ $message-border: $border-width solid $border !default;
 			background-color: $message-sender-background-color;
 		}
 	}
+
+	&.message-size-sm {
+		font-size: $message-font-size-sm;
+
+		.persona-details .persona-name {
```

Comment:
You should not modify one component inside another one. Persona has [size variations](https://design.learn.microsoft.com/pulls/581/components/persona.html#sizes) as well, if needed just use the `persona-sm` in the markup,

Reply:
Sure. Removed.